scripts/install

```diff
@@ -99,7 +99,7 @@ chown $app:$app "$install_dir/config/local-production.json"
 ynh_script_progression --message="Starting a systemd service..."
 
 # Start a systemd service
-ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" --line_match="HTTP server listening on 127.0.0.1"
+ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" #--line_match="HTTP server listening on 127.0.0.1"
```

Comment:
Why removing that service startup confirmation ? 🤔

Reply:
only for testing purpose...

Reply:
Oh ok. I was confused by the "cleaning" label of the commit.
The LDAP plugin install issue is very likely to be caused by this comment, as the database might not be ready when the installation starts. As seen in #400